solr/datadog_checks/solr/data/conf.yaml.example

```diff
@@ -25,6 +25,142 @@ init_config:
   # Agent 5: Customize all your metrics below
   # Agent 6: The default metrics to be collected are kept in metrics.yaml, but you can still add your own metrics here
   conf:
+
+
+  # Solr 7 MBeans:
+  - include:
+      category: SEARCHER
+      name:
+        - numDocs
+      scope:
+        - searcher
+      attribute:
+        Value:
+          alias: solr.searcher.numdocs
```

Comment:
Also, why aren't all metrics listed in the metadata file?

Reply:
All metrics should be in the metadata.csv file, let me know if I've missed any. 
`solr.searcher.numdocs` is here - https://github.com/DataDog/integrations-core/blob/c23d805f6aa4e306e552554a763b2f80724b2c0c/solr/metadata.csv#L3

Reply:
ah ok. I only look at the diff. I didn't see others were there already.